Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/util/StringUtil.java

```diff
@@ -44,10 +45,13 @@
     private static final char[] INVALID_CHARS = {'\\', '/', ':', '*', '?', '"', '<', '>', '|', '[', ']', '\'', ';',
             '=', ','};
     private static final char SANITIZED_CHAR = '_';
+    private static final int FILE_NAME_MAX_LENGTH = 255;
 
-    private static final String dropboxPrefix = "https://www.dropbox.com";
-    private static final String googleDrivePrefix = "https://drive.google.com";
-    private static final String oneDrivePrefix = "https://onedrive.live.com";
+    private static final String VALID_PROTOCOL = "https://";
+
+    private static final String DROPBOX_DOMAIN = "www.dropbox.com";
```

Comment:
add this to a list and check with contains

Reply:
The reason to have separate domains is they are used in other places to identify URL of different domains and convert from sharable URL to downloadable URL.